applications/matlab-batch/app.json

```diff
@@ -0,0 +1,101 @@
+{
+  "id": "matlab-batch",
+  "version": "23.2",
+  "description": "MATLAB Batch is a non-interactive application to run *.m scipts.  Submit a help ticket to get your license configured or approved.",
+  "owner": "${apiUserId}",
+  "runtime": "ZIP",
+  "runtimeVersion": null,
+  "runtimeOptions": null,
+  "containerImage": "tapis://cloud.data/corral/tacc/aci/CEP/applications/v3/matlab-batch/2023b/matlab-batch.zip",
+  "jobType": "BATCH",
+  "maxJobs": -1,
+  "maxJobsPerUser": -1,
+  "strictFileInputs": true,
+  "jobAttributes": {
+    "description": "",
+    "dynamicExecSystem": false,
+    "execSystemConstraints": null,
+    "execSystemId": "stampede3",
+    "execSystemExecDir": "${JobWorkingDir}",
+    "execSystemInputDir": "${JobWorkingDir}",
+    "execSystemOutputDir": "${JobWorkingDir}",
+    "execSystemLogicalQueue": "skx-dev",
+    "archiveSystemId": "cloud.data",
+    "archiveSystemDir": "HOST_EVAL($HOME)/tapis-jobs-archive/${JobCreateDate}/${JobName}-${JobUUID}",
+    "archiveOnAppError": true,
+    "isMpi": false,
+    "mpiCmd": null,
+    "cmdPrefix": null,
+    "parameterSet": {
+      "appArgs": [],
+      "containerArgs": [],
+      "schedulerOptions": [
+        {
+          "name": "TACC Scheduler Profile",
+          "description": "Scheduler profile for MATLAB Batch",
+          "inputMode": "FIXED",
+          "arg": "--tapis-profile matlab-2023b",
+          "notes": {
+            "isHidden": true
+          }
+        },
+        {
+          "name": "Slurm job name",
+          "description": "Set the slurm job name to be identical to the Tapis job name.",
+          "inputMode": "FIXED",
+          "arg": "--job-name ${JobName}",
+          "notes": {
+            "isHidden": true
+          }
+        }
+      ],
+      "envVariables": [
+        {
+          "key": "matlabScriptName",
+          "value": "",
+          "description": "Provide the filename of the MATLAB script to execute, relative to the path of the Working Directory. This file should reside in the Working Directory specified. Example: 'mymatlabtest.m'.",
+          "inputMode": "REQUIRED",
+          "notes": {
+            "label": "MATLAB Script Filename"
+          }
+        }
+      ],
+      "archiveFilter": {
+        "includes": [],
+        "excludes": [
+          "matlab-batch.zip"
+        ],
+        "includeLaunchFiles": true
+      }
+    },
+    "fileInputs": [
+      {
+        "name": "Working Directory",
+        "description": "The directory containing the files that you want to work on. This directory and its files will be copied to where your interactive session runs. Click the 'Select Input' button and then select the directory you wish to work from.",
```

Comment:
We are trying to maintain a sample app input folder at /corral-repl/tacc/aci/CEP/community/app_examples/ 
You can add your test folder this.

Reply:
Done!